server/src/graql/query/pattern/Pattern.java

```diff
@@ -65,19 +65,18 @@
     @CheckReturnValue
     Disjunction<Conjunction<Statement>> getDisjunctiveNormalForm();
 
+    @CheckReturnValue
+    Disjunction<Conjunction<Pattern>> getNegationDNF();
+
     /**
-     * @return this pattern negated
+     * @return true if this pattern corresponds to a negation pattern
      */
     @CheckReturnValue
-    Pattern negate();
+    default boolean isNegation(){ return false; }
```

Comment:
We don't need this method as it does exactly the same thing as `instanceof Negation`.